### ISSUE TITLE
Vagrant up fails

### ISSUE THREAD

**Issue by codeyy** [geerlingguy/drupal-vm#802]
I'm completely new to this so I tried my best to follow the manual, re-read it, look for errors, but I couldn't find my issue.

I'm running Windows 10 Pro 64-bit, with VirtualBox 5.0.24 freshly installed (after 5.1 caused problems), and with Vagrant 1.8.4 and DrupalVM 3.1.4.

> C:\DrupalVM>vagrant up
> Bringing machine 'drupalvm' up with 'virtualbox' provider...
> ==> drupalvm: Box 'geerlingguy/ubuntu1604' could not be found. Attempting to find and install...
>     drupalvm: Box Provider: virtualbox
>     drupalvm: Box Version: >= 0
> The box 'geerlingguy/ubuntu1604' could not be found or
> could not be accessed in the remote catalog. If this is a private
> box on HashiCorp's Atlas, please verify you're logged in via
> `vagrant login`. Also, please double-check the name. The expanded
> URL and error message are shown below:
> 
> URL: ["https://atlas.hashicorp.com/geerlingguy/ubuntu1604"]
> Error:
> 
> C:\DrupalVM>

This is a command prompt running as Administrator.
I tried switching to ubuntu1404, but that gave me the same error.

**Comment by fubarhouse** [comment]
What's the output from this?

    vagrant box add geerlingguy/ubuntu1604 https://atlas.hashicorp.com/geerlingguy/boxes/ubuntu1604/versions/1.0.4/providers/virtualbox.box

**Comment by codeyy** [comment]
> C:\HashiCorp\Vagrant>vagrant box add geerlingguy/ubuntu1604 https://atlas.hashicorp.com/geerlingguy/boxes/ubuntu1604/versions/1.0.4/providers/virtualbox.box
> ==> box: Box file was not detected as metadata. Adding it directly...
> ==> box: Adding box 'geerlingguy/ubuntu1604' (v0) for provider:
>     box: Downloading: https://atlas.hashicorp.com/geerlingguy/boxes/ubuntu1604/versions/1.0.4/providers/virtualbox.box
>     box:
> An error occurred while downloading the remote file. The error
> message, if any, is reproduced below. Please fix this error and try
> again.
> 
> C:\HashiCorp\Vagrant>

**Comment by fubarhouse** [comment]
I'm afraid I can't help much more, but it definitely seems like a windows-based problem - and I've never used vagrant via windows.

You can read more, and some potential solutions here:

http://stackoverflow.com/questions/34463828/vagrant-cant-download-box

I understand others here might be able to help, but I have tried :)

Feel free to chip in people!

**Comment by thom8** [comment]
@codeyy take a look at -- http://docs.drupalvm.com/en/latest/other/windows/#command-line-environment

**Comment by codeyy** [comment]
@thom8 that page gives away a lot of tips and tricks, thanks.

I downloaded cmder, since it has built-in GIT and SSH support it was a no brainer. It seems to me that cygwin has more possibilities, but for my needs cmder is the best fit solution.
That resulted in an error that was displayed beautiful in red, but the same error...

The "Synced Folders" issues talked about all seem unrelated.

So I'm afraid nothing of that worked...


@fubarhouse I started following your tips and tricks after that.

From the stackoverflow page I tried a few things :
1. I installed "Microsoft Visual C++ 2010 SP1 Redistributable Package (x64)", same error...
2. I downloaded the latest version of curl.exe and put it in \Vagrant\embedded\bin, and that did the trick! 

Vagrant is doing it's thing now... *fingers crossed*
